design.config.js

```diff
@@ -0,0 +1,37 @@
+const path = require('path')
+
+const isDev = process.env.NODE_ENV !== 'production'
```

Comment:
Is there a way to drop this condition from the config file? I mean conditional icons adding. I am afraid it will cause problems.

Reply:
I don't know. I mean I didn't like it either but that was the only solution I could find.

Reply:
I think that we should build a custom library version for the documentation, i.e. build the documentation with the library as a dependency and have a separate configuration there (with icons needed for the documentation examples).

However, using some CDN seems to be even better to me because it may work even it markdown.

Let's keep this out of scope at the moment.